testing/src/test/java/io/stargate/it/storage/ExternalStorage.java

```diff
@@ -32,13 +33,15 @@
 import org.slf4j.LoggerFactory;
 
 /** JUnit 5 extension for tests that need a backend database cluster managed by {@code ccm}. */
-public class ExternalStorage implements BeforeAllCallback, AfterAllCallback, ParameterResolver {
+public class ExternalStorage
+    implements BeforeAllCallback, AfterAllCallback, ParameterResolver, BeforeTestExecutionCallback {
 
   private static final Logger LOG = LoggerFactory.getLogger(ExternalStorage.class);
 
   private static final String CCM_VERSION = "ccm.version";
   private static final boolean EXTERNAL_BACKEND =
       Boolean.getBoolean("stargate.test.backend.use.external");
+  private static final String DC_OVERRIDE = System.getProperty("stargate.test.backend.dc");
```

Comment:
This is added "in case we need it later", right? I don't see it being used otherwise in the PR.

Reply:
No, if you run `ccm` locally like `ccm create -n 1 --dse ...` the cluster name is going to be `Cassandra`, so the option is quite useful for local runs with a permanent DSE node.

Reply:
Using the `-n 1:0` (or `-n 3:0`) ccm forces the dc to be `"dc1"`:

```
ccm create test -v 3.11.6 -n 1:0 -s -b
```

```
ccm create test --dse -v 6.8.0 -n 1:0 -s -b
```

Reply:
I would keep property `stargate.test.backend.dc` with dc1 as default (regardless of dse/C*).

Reply:
> ccm create test --dse -v 6.8.0 -n 1:0 -s -b

right, but I also run local DSE builds often, and they have DC name = `Cassandra` by default :)

Reply:
I will refactor this property with `dc1` default

Reply:
updated and force pushed